.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@
 name: CI
 
 # Controls when the action will run. Triggers the workflow on push or pull request events on every branch
-on: [ push, pull_request ]
+on: [push, pull_request]
```

Comment:
Just to be sure: So we don't want extra spaces on the inside of brackets?

Reply:
I think both options are okay, but that's what the VSCode default yaml formatter suggested…